packages/feedback/src/screenshot/components/ScreenshotEditor.tsx

```diff
@@ -407,113 +164,19 @@ export function ScreenshotEditorFactory({
-                    top={croppingRect.startY - CROP_BUTTON_BORDER}
-                    onGrabButton={onGrabButton}
-                    corner="top-right"
-                  ></CropCorner>
-                  <CropCorner
-                    left={croppingRect.startX - CROP_BUTTON_BORDER}
-                    top={croppingRect.endY - CROP_BUTTON_SIZE + CROP_BUTTON_BORDER}
-                    onGrabButton={onGrabButton}
-                    corner="bottom-left"
-                  ></CropCorner>
-                  <CropCorner
-                    left={croppingRect.endX - CROP_BUTTON_SIZE + CROP_BUTTON_BORDER}
-                    top={croppingRect.endY - CROP_BUTTON_SIZE + CROP_BUTTON_BORDER}
-                    onGrabButton={onGrabButton}
-                    corner="bottom-right"
-                  ></CropCorner>
-                </div>
-              )}
-              {isCropping && (
-                <div
-                  style={{
-                    left: Math.max(0, croppingRect.endX - 191),
-                    top: Math.max(0, croppingRect.endY + 8),
-                  }}
-                  class={`editor__crop-btn-group ${confirmCrop ? 'editor__crop-btn-group--active' : ''}`}
-                >
-                  <button
-                    onClick={e => {
-                      e.preventDefault();
-                      if (croppingRef.current) {
-                        setCroppingRect({
-                          startX: 0,
-                          startY: 0,
-                          endX: croppingRef.current.width / DPI,
-                          endY: croppingRef.current.height / DPI,
-                        });
-                      }
-                      setConfirmCrop(false);
-                    }}
-                    class="btn btn--default"
-                  >
-                    {options.cancelButtonLabel}
-                  </button>
-                  <button
-                    onClick={e => {
-                      e.preventDefault();
-                      applyCrop();
-                      setConfirmCrop(false);
-                    }}
-                    class="btn btn--primary"
-                  >
-                    {options.confirmButtonLabel}
-                  </button>
-                </div>
-              )}
-            </div>
-            <canvas
-              class={`editor__annotation ${isAnnotating ? 'editor__annotation--active' : ''}`}
-              onMouseDown={onAnnotateStart}
-              ref={annotatingRef}
-            ></canvas>
+            <Crop
+              action={action}
+              imageBuffer={imageBuffer}
+              croppingRef={croppingRef}
+              cropContainerRef={cropContainerRef}
+              croppingRect={croppingRect}
+              setCroppingRect={setCroppingRect}
+              resize={resize}
+            />
+            <Annotations action={action} imageBuffer={imageBuffer} annotatingRef={annotatingRef} />
```

Comment:
Would it work if this was like:
```suggestion
            {action === 'annotate' ? <Annotations imageBuffer={imageBuffer} annotatingRef={annotatingRef} /> : null}
```

same for `action === 'crop' ? <Crop> : null`

Reply:
no it does not :(